src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesProvisioningLimits.java

```diff
@@ -121,13 +120,11 @@ public void unregister(@Nonnull KubernetesCloud cloud, @Nonnull PodTemplate podT
     }
 
     @Nonnull
-    @VisibleForTesting
```

Comment:
Can we keep something like // VisibleForTesting ? This has still documentation value.

Reply:
Just wondering if there was an alternative java annotation or access modifier we can use?

Reply:
Well, we could maybe use something like `@Restricted(NoExternalUse.class)`